src/gateways/googleAnalyticsClient.ts

```diff
@@ -196,18 +213,20 @@ class GoogleAnalyticsClient {
       return;
     }
 
-    for (const tagId of this.tagIds) {
+    for (const tag of this.tags) {
       command.payload = {...command.payload, queue_time: queueTime };
+
       if (command.name === 'event') {
-        this.gtag('event', command.eventName, { ...command.payload, send_to: tagId });
+        if (tag.ignoredEventNames.includes(command.eventName)) { return; }
```

Comment:
I think this should be `continue` instead?

Reply:
But then maybe you need some code to abort if everything got skipped

Reply:
Or maybe no need to check for the abort condition, since there's nothing after the loop.

Reply:
Whoops good catch, that was definitely a typo and should have been `continue`